Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -2555,16 +2532,31 @@ def long_running_main():
     Starts the long running thread.
     """
     try:
-        asyncio.run(start_listening(), debug=True)
+        start_listening()
     except Exception as e:
         demisto.error(f"The Loop has failed to run {str(e)}")
-    finally:
-        loop = asyncio.get_running_loop()
+
+
+def fetch_incidents():
+    """
+    Used to offload processing into a background task
+    This will run every 15 seconds. For fetch intervals of 1 minute, this should execute 4 times.
+    """
+    interval = 0
+    while interval < 3:
         try:
-            loop.stop()
-            loop.close()
-        except Exception as e_:
-            demisto.error(f'Failed to gracefully close the loop - {e_}')
+            if MIRRORING_ENABLED:
+                check_for_mirrors()
+            check_for_unanswered_questions()
+            if EXTENSIVE_LOGGING:
+                stats, _ = slack_get_integration_context_statistics()
+                demisto.debug(f'Integration Context Stats\n_____________\n{stats}')
+            interval += 1
+            time.sleep(15)
+        except Exception as e:
+            demisto.error(f'An error occurred: {e}')
```

Comment:
Better to make the log messages unique. e.g.
```suggestion
            demisto.error(f'An error occurred while fetching incidents: {e}')
```